js/shopping_list_test.js

```diff
@@ -0,0 +1,118 @@
+/* jshint esversion:6*/
+const expect = chai.expect;
+
+describe('Shopping List Item Class', function(){
+  var listItem = ShoppingListItem;
+  var someName = null;
+  before(function(){
```

Comment:
You generally don't want tests to be self referencing. If you set the Item.name your test should test the value you set to .name not .name itself.

```
let testName = 'carrot';
let testDescription = 'a vegetable';
new  ShoppingListItem(testName, testDescription);

```